test/conftest.py

```diff
@@ -31,6 +31,104 @@ def target():
     return _start_proc
 
 
+@pytest.fixture(scope="session", autouse=True)
+def node_infos():
+    from . import IDS, PORTS
+
+    return zip(IDS, PORTS)
+
+
+@pytest.fixture(scope="session", autouse=True)
+def init_gateway():
+    import time
+
+    def setUpGateway(port):
+        import os
```

Comment:
Can these inner imports be moved to the beginning of the file?

Reply:
Done! (4c90afd )